assets/schemas/manifest.json

```diff
@@ -696,121 +1319,284 @@
         }
       }
     },
-    "form": {
-      "type": ["array", "null"],
+    "purposeTemplates": {
+      "type": "array",
       "items": {
         "type": "object",
-        "required": [
-          "name",
-          "type"
-        ],
-        "additionalProperties": false,
         "properties": {
+          "code": {
+            "type": "string"
+          },
           "name": {
-            "type": "string",
-            "description": "The key field is where the data will be saved to. This must be unique per field.\n For example, if key = 'customers' then the value of the field will be saved in data.customers"
+            "type": "string"
           },
-          "title": {
-            "type": "string",
-            "description": "The name or title for this component"
+          "description": {
+            "type": "string"
           },
-          "type": {
+          "tcfId": {
```

Comment:
what does this correspond to?

Reply:
The TCF purposes have an ID in the Global Vendor List (GVL). This is how we link from here to the GVL.

Reply:
ah right